### PR TITLE
Add Configuration File for Workflow Recommendations

### PR DESCRIPTION
**Pull request by yangbobo2021**
## **User description**
This PR introduces a new `configuration.toml` file to the workflows plugin, aimed at displaying recommended workflows as proposed in issue [#304](https://github.com/devchat-ai/devchat/issues/304). The file has been populated with an initial set of recommended workflows and is formatted to be properly readable using a provided Python script to verify its correctness.

**Key Changes:**
- A `configuration.toml` file was added containing the recommended workflows.
- The file's format has been verified to be correct using a Python script for reading and printing out the workflow configuration data.

close https://github.com/devchat-ai/devchat/issues/304
The integrity of the configuration file was confirmed by reading its content with the following Python script:

```python
import toml

def read_recommended_workflows(filepath):
try:
config = toml.load(filepath)
recommended_workflows = config.get("recommend", {}).get("workflows", [])
return recommended_workflows
except Exception as e:
print(f"Error reading config file: {e}")
return []

config_filepath = 'configuration.toml'
workflows = read_recommended_workflows(config_filepath)

print("Recommended workflows include:")
for workflow in workflows:
print(workflow)
```

___

## **Type**
enhancement

___

## **Description**
- Introduced a `configuration.toml` file to specify recommended workflows.
- The `recommend` section within the file lists initial recommended workflows: unit_tests, commit, docstring, comment, explain, fix.
- This setup aims to enhance the workflows plugin by providing a structured recommendation system.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>configuration.toml</strong><dd><code>Introduce Recommended Workflows Configuration</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
configuration.toml

<li>Added a new section <code>recommend</code> with a key <code>workflows</code>.<br> <li> Populated <code>workflows</code> with an array of recommended workflows: <br>unit_tests, commit, docstring, comment, explain, fix.<br>

</details>
    

  </td>
  <td><a href="https://github.com/devchat-ai/workflows/pull/81/files#diff-f5da3be18d2f55a19a3e5773f027adcc3df5a569e33b30e1b64104350a171c14">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: configuration.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[recommend]
+workflows = ["unit_tests", "commit", "docstring", "comment", "explain", "refactor"]
```
Comment (kagami-l) [inline configuration.toml:2]: 为什么项目里有的配置文件用yaml、有的用toml？ @yangbobo2021

Comment (yangbobo2021) [reply configuration.toml:2]: 本来这里想用yaml来着，但看了下其他项目现在用toml会很多，例如poetry、pr-agent，感觉这个会是未来的趋势，所以就选择了大多数人的选择，这里就使用了toml格式。

**Review by small-stone — APPROVED** [review]

**Review by small-stone — APPROVED** [review]

**Review by pplam — APPROVED** [review]

**Pull request merged**